src/components/SelectBoxSearch.js

```diff
@@ -155,7 +155,7 @@ class SelectBoxSearch extends Component {
               return (
                 <div
                   tabIndex={i + 1}
-                  key={i}
+                  key={Math.floor(Math.random() * 100000)}
```

Comment:
Dit zijn generieke componenten.
Hiervoor is het dus eigelijk niet duidelijk wat de unieke key zou moeten zijn.
Als workaround gebruik ik dus nu een random generator, maar eigelijk zou er in de props misschien een functie moeten zijn die van een item een unieke key maakt.

Reply:
Bijv index + currentChoiceString ?